spec/features/admin/members/admin_search_member_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+feature 'Admin can search for users by email' do
+
+  signed_in_as(:admin) do
+    let!(:member) { FactoryGirl.create(:user, :member) }
+
+    before do
+      click_header_option("Dashboard")
+      click_sidemenu_option("Members")
+    end
+
+    scenario "Search with matching data" do
+      fill_in('User email search', :with => member.email)
+      click_button('SEARCH')
+      expect(page).to have_content(member.email)
+      expect(page).not_to have_content(current_user.email)
+    end
+
+    scenario "Search with non-matching data" do
+      fill_in('User email search', :with => current_user.email)
+      click_button('SEARCH')
+      expect(page).to have_content("No users matched that search")
+      expect(page).not_to have_content(current_user.email)
+    end
+  end
+
+end
```

Comment:
Add newline at the end of the file.

[Some stuff about it here](https://robots.thoughtbot.com/no-newline-at-end-of-file)

Add `"ensure_newline_at_eof_on_save option": true` to preferences